packages/icons/src/align-self-baseline.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
```

Comment:
we started generating icon components out of SVG, check out the package scripts

Reply:
cc @TrySound

Reply:
Props are spread and ref is added. See generated icons. Fill on the root should be enough. I dont see the issue.